gallery/src/components/NxSmallVulnerabilityCounter/NxSmallVulnerabilityCounterPage.tsx

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxTextLink, NxTable, NxP, NxCode } from '@sonatype/react-shared-components';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import NxSmallVulnerabilityCounterExample from './NxSmallVulnerabilityCounterExample';
+import NxSmallVulnerabilityCounterCustomExample from './NxSmallVulnerabilityCounterCustomExample';
+import NxSmallVulnerabilityCounterInfiniteMaxDigitsExample from './NxSmallVulnerabilityCounterInfiniteMaxDigitsExample';
+
+const NxSmallVulnerabilityCounterCode = require('./NxSmallVulnerabilityCounterExample?raw');
+const NxSmallVulnerabilityCounterCustomCode = require('./NxSmallVulnerabilityCounterCustomExample?raw');
+const NxSmallVulnerabilityCounterInfiniteMaxDigitsCode = require(
+    './NxSmallVulnerabilityCounterInfiniteMaxDigitsExample?raw');
+
+const NxSmallVulnerabilityCounterPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
+        Similar to <NxCode>NxVulnerabilityCounter</NxCode>, <NxCode>NxSmallVulnerabilityCounter</NxCode>
+        displays a series of counts of "Vulnerabilities" in different severity ratings. This component
```

Comment:
Missing rendered whitespace before "displays"

Reply:
Rendered the whitespace here: [80678ec](https://github.com/sonatype/sonatype-react-shared-components/pull/1200/commits/80678ec33279cd9c87ce634a739d5aced17284ba)
I also added a console.log so we don't have the error you mentioned on https://github.com/sonatype/sonatype-react-shared-components/pull/1200#issuecomment-2407700279 but please let me know if that's a wrong approach for this and then I can change it on the feature branch

Reply:
> I also added a console.log so we don't have the error you mentioned on https://github.com/sonatype/sonatype-react-shared-components/pull/1200#issuecomment-2407700279, the comment on the code suggests to eslint-disable-next-line @typescript-eslint/no-unused-vars but please let me know if that's a wrong approach for this and then I can change it on the feature branch

That's fine except that it causes its own eslint warning that you need to disable:

```
/home/rpokorny/Documents/sonatype-react-shared-components/gallery/src/jsUtilPages/VulnerabilityLevelUtils/VulnerabilityTypesAndRatingsExample.ts
  17:1  warning  Unexpected console statement  no-console

✖ 1 problem (0 errors, 1 warning)
```

Reply:
> > I also added a console.log so we don't have the error you mentioned on [#1200 (comment)](https://github.com/sonatype/sonatype-react-shared-components/pull/1200#issuecomment-2407700279), the comment on the code suggests to eslint-disable-next-line @typescript-eslint/no-unused-vars but please let me know if that's a wrong approach for this and then I can change it on the feature branch
> 
> That's fine except that it causes its own eslint warning that you need to disable:
> 
> ```
> /home/rpokorny/Documents/sonatype-react-shared-components/gallery/src/jsUtilPages/VulnerabilityLevelUtils/VulnerabilityTypesAndRatingsExample.ts
>   17:1  warning  Unexpected console statement  no-console
> 
> ✖ 1 problem (0 errors, 1 warning)
> ```

Disabled here: [a22c9d6](https://github.com/sonatype/sonatype-react-shared-components/pull/1200/commits/a22c9d634f382a9d43c09f529eb59ae2eaeb3704)